deepdrivemd/agents/stream/dbscan.py

```diff
@@ -133,15 +133,15 @@ def predict(
 ) -> np.ndarray:
     """Project contact maps into the middle layer of CVAE
 
-    Parameters:
-    -------
+    Parameters
+    --------------
```

Comment:
The ---------- syntax should be equal in length to the line above e.g.:

```
Parameters
----------

Returns
-------

Raises
------
```
...

Reply:
The specifications allow the line to be longer than the text. Sphinx complains only if it is shorter. Or do you want some extra style constraints?

Reply:
Yes, extra style constraints to obey the numpydocstyle. I'm actually surprised it is not auto-linted, maybe I need to add a setting for that.

Reply:
Done